cms/static/js/spec/views/video_thumbnail_spec.js

```diff
@@ -149,38 +186,152 @@ define(
+                expect(videoThumbnailView.readMessages).toHaveBeenCalledWith(
+                    ['Could not upload the video image file', 'This image file type is not supported.']
+                );
+            });
+
+            it('should show error message in case of server error', function() {
+                var videoListView = createVideoListView(),
+                    $videoListEl = videoListView.render().$el,
+                    videoThumbnailView = videoListView.itemViews[0].videoThumbnailView, // eslint-disable-line no-shadow
+                    $el = videoThumbnailView.render().$el,
                     requests = AjaxHelpers.requests(this);
 
                 videoThumbnailView.chooseFile();
 
                 // Add image to upload queue and send POST request to upload image
-                $el.find('.upload-image-input').fileupload('add', {files: [createFakeImageFile(60)]});
+                $el.find('.upload-image-input').fileupload('add', {files: [createFakeImageFile()]});
 
                 AjaxHelpers.respondWithError(requests);
 
-                expect($('#notification-error-title').text().trim()).toEqual(
-                    "Studio's having trouble saving your work"
-                );
+                // Verify error message is present
+                expect($videoListEl.find('.thumbnail-error-wrapper')).toExist();
             });
 
-            it('calls readMessage with correct message', function() {
-                spyOn(videoThumbnailView, 'readMessages');
+            it('should show error message when file is smaller than minimum size', function() {
+                var errorMessage,
+                    $thumbnailErrorEl,
+                    videoListView = createVideoListView(),
+                    $videoListEl = videoListView.render().$el,
+                    videoThumbnailView = videoListView.itemViews[0].videoThumbnailView, // eslint-disable-line no-shadow
+                    $el = videoThumbnailView.render().$el;
 
-                videoThumbnailView.imageSelected({}, {submit: function() {}});
-                expect(videoThumbnailView.readMessages).toHaveBeenCalledWith(['Video image upload started']);
-                videoThumbnailView.imageUploadSucceeded({}, {result: {image_url: UPLOADED_IMAGE_URL}});
-                expect(videoThumbnailView.readMessages).toHaveBeenCalledWith(['Video image upload completed']);
-                videoThumbnailView.imageUploadFailed();
-                expect(videoThumbnailView.readMessages).toHaveBeenCalledWith(['Video image upload failed']);
+                videoThumbnailView.chooseFile();
+
+                // Add image to upload queue and send POST request to upload image
+                $el.find('.upload-image-input')
+                    .fileupload('add', {files: [createFakeImageFile(VIDEO_IMAGE_MIN_BYTES - 10)]});
+
+                errorMessage = 'The selected image must be larger than ' +
```

Comment:
@muhammad-ammar I am verifying meesage from the view below, this is just a a variable to hold expected error message